docs/fidesops/docs/saas_connectors/saas_connectors.md

```diff
@@ -50,7 +50,7 @@ PATCH api/v1/connection/{saas_key}/saas_config
 ```
 3. Configure the secrets. The SaaS config must already defined to provide validation for the secrets.
 ```
-PUT api/v1/connection/{saas_key}/secret?verify=true
```

Comment:
I refer to these instructions from the OAuth documentation and we can't verify right away for OAuth2 connectors so I removed `verify=true` so these instructions apply to both OAuth2 and non-OAuth2 connectors.